mq/publishers.go

```diff
@@ -23,7 +23,7 @@ func PublishPeerUpdate(newNode *models.Node) error {
 	}
 	for _, node := range networkNodes {
 
-		if node.IsServer == "yes" {
+		if node.IsServer == "yes" || newNode.ID == node.ID { // skip servers and self
```

Comment:
with this change, a newly joined node will not receive a peer update to up to 5 minutes.